pkg/bucket/bucket.go

```diff
@@ -113,17 +114,17 @@ func RunCreate(cmd *cobra.Command, args []string) {
 	maxSize, _ := cmd.Flags().GetString("max-size")
 	maxObjects, _ := cmd.Flags().GetString("max-objects")
 
-	err = nbClient.CreateBucketAPI(nb.CreateBucketParams{Name: bucketName, Tiering: tierName, ForceMd5Etag: forceMd5Etag})
+	quota, updateQuota, err := prepareQuotaConfig(bucketName, maxSize, maxObjects)
 	if err != nil {
-		log.Fatal(err)
+		log.Fatalf(`❌ Could not create bucket "%q" validation failed %q`, bucketName, err)
 	}
 
-	quota, updateQuota := prepareQuotaConfig(maxSize, maxObjects)
-	if updateQuota {
+	err = nbClient.CreateBucketAPI(nb.CreateBucketParams{Name: bucketName, Tiering: tierName, ForceMd5Etag: forceMd5Etag})
+	if err == nil && updateQuota {
 		err = nbClient.UpdateBucketAPI(nb.CreateBucketParams{Name: bucketName, Quota: &quota})
```

Comment:
Not sure why we havn't added quota in create_bucket api, so we need to call update_bucket  api with quota again here.

@vh05 @liranmauda Anything you want to add?

Reply:
I think we have not introduce there because we fully not supporting quota. @liranmauda @dannyzaken Please correct me if wrong